org.agileware.natural.cucumber/src/org/agileware/natural/cucumber/serializer/CucumberSerializer.xtend

```diff
@@ -1,42 +1,52 @@
 package org.agileware.natural.cucumber.serializer
 
+import com.google.inject.Inject
+import org.agileware.natural.cucumber.cucumber.AbstractScenario
 import org.agileware.natural.cucumber.cucumber.Background
-import org.agileware.natural.cucumber.cucumber.DocString
+import org.agileware.natural.cucumber.cucumber.CucumberModel
 import org.agileware.natural.cucumber.cucumber.Example
 import org.agileware.natural.cucumber.cucumber.Feature
 import org.agileware.natural.cucumber.cucumber.Scenario
 import org.agileware.natural.cucumber.cucumber.ScenarioOutline
 import org.agileware.natural.cucumber.cucumber.Step
-import org.agileware.natural.cucumber.cucumber.Table
-import org.agileware.natural.cucumber.cucumber.TableCol
-import org.agileware.natural.cucumber.cucumber.TableRow
-import org.agileware.natural.cucumber.cucumber.Tag
+import org.agileware.natural.lang.serializer.NaturalSerializer
 import org.eclipse.xtext.nodemodel.util.NodeModelUtils
-import org.agileware.natural.cucumber.cucumber.Text
 
 class CucumberSerializer {
```

Comment:
`CucumberSeriaizer` is out of date and needs to be updated to reflect updates to the AST. This is technically only used for debugging purposes at the moment, so not a huge deal. It will be required for two-way serialization to work however.